.eslintrc

```diff
@@ -118,7 +118,7 @@
     "no-implied-eval": 2,
     "no-iterator": 2,
     "no-lone-blocks": 2,
-    "no-mixed-operators": 1,
```

Comment:
Do we need to update strict elsint, too?

Reply:
It doesn't exist.